src/components/questionComponent.js

```diff
@@ -1,14 +1,13 @@
-import { ANSWERS_LIST_ID } from '../constants.js';
-import { NEXT_QUESTION_BUTTON_ID } from '../constants.js';
-
 /**
  * Create a full question element
- * @returns {Element}
+ * @param {string} question - The text of the question.
+ * @returns {HTMLDivElement} - The full question element
  */
+import { ANSWERS_LIST_ID } from '../constants.js';
+import { NEXT_QUESTION_BUTTON_ID } from '../constants.js';
```

Comment:
Imports always should be set in the very top of every file!

Reply:
updated